google-cloud-datastore/lib/google/cloud/datastore/v1/doc/overview.rb

```diff
@@ -17,7 +17,7 @@ module Cloud
     # rubocop:disable LineLength
 
     ##
-    # # Ruby Client for Google Cloud Datastore API ([Alpha](https://github.com/GoogleCloudPlatform/google-cloud-ruby#versioning))
+    # # Ruby Client for Google Cloud Datastore API ([GA](https://github.com/GoogleCloudPlatform/google-cloud-ruby#versioning))
```

Comment:
This file is a generated GAPIC file, and we have typically not modified them by hand.

@geigerj, should we modify this file? Or should we regenerate all the GAPIC files?

Reply:
Ah right, missed the path, thanks @blowmage!

Reply:
Ack. I can file a new PR, or push a commit to this one if you give me permissions on your branch.

Reply:
I'll close this out and you can move forward with your PR. Thanks @geigerj !